packages/mobile/src/import/saga.ts

```diff
@@ -29,7 +29,7 @@ export function* importBackupPhraseSaga({ phrase, useEmptyWallet }: ImportBackup
   Logger.debug(TAG + '@importBackupPhraseSaga', 'Importing backup phrase')
   yield call(waitWeb3LastBlock)
   try {
-    if (!validateMnemonic(phrase, undefined, bip39)) {
+    if (!validateMnemonic(phrase)) {
```

Comment:
If you avoid passing the bip39 lib, will use the one from the utils, instead of the `react-native` one.
For what I know, it has some optimisations. I should recommend to keep this line as it was

Reply:
That's a good point. I didn't notice we were using a library optimized for mobile. I will add this back in.